DesignSafe-CI/portal
# des-2329-removed-item-list-bottom-margin

## Overview: ##
Fix Data Depot's left-hand navigation hover bug. When hovering over the different directories in the Data Depot, everything shifts around.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2329](https://jira.tacc.utexas.edu/browse/DES-2329)

## Summary of Changes: ##
removed the bottom margin on the item list which displays the left navigation bar.

## Testing Steps: ##
1. Click through the different option on the left nav bar and see that the options don't move anymore when you hover over them. 

## UI Photos:

## Notes: ##
